narayana-bom/pom.xml

```diff
@@ -16,6 +16,13 @@
   <name>Narayana Bom</name>
   <description>Narayana BOM</description>
 
+<!--
+    This BOM is for developers' use across the Narayana repos rather than end users'.
+    This BOM's purpose is to collect all dependencies (with default scope, which is 'provided') needed in Narayana repos and centralize their update
+    In this way when a dependency is updated it is updated in all Narayana repositories
+    and we could immediately test the change everywhere and evaluate its impact 
+-->
+
```

Comment:
Thanks Marco. WDYT if we go for something shorter? Something like:
`This Bill of Materials (BOM) is for centralising dependency management across the various modules of Narayana.`

Reply:
Yes, I think something shorter would be better. I would underline anyway that this is for " developers' use across the Narayana repos rather than end users'." What about keeping that too?